lightning/src/ln/channelmanager.rs

```diff
@@ -9022,8 +9040,7 @@ where
 			.collect::<Vec<_>>();
 
 		self.router
-			.create_blinded_paths(recipient, peers, secp_ctx)
-			.and_then(|paths| paths.into_iter().next().ok_or(()))
```

Comment:
Move this to the calls sites where we only want one path. Then you don't need to change `OfferBuilder`. We may want to return an iterator, too, to avoid unnecessary `Vec` allocations.

Reply:
Done!
Move the `and_then` call to call site, and revert the `path` functions in [4032289](https://github.com/lightningdevkit/rust-lightning/commit/40322898323c2656779c1ff23e54310229ecd671)
Thanks!